packages/storage/src/indexed-db/index.ts

```diff
@@ -469,6 +473,29 @@ export class IndexedDb implements IndexedDbInterface {
     return epoch;
   }
 
+  /**
+   * Inserts the validator info into the database, or updates an existing
+   * validator info if one with the same identity key exists.
+   */
+  async upsertValidatorInfo(validatorInfo: ValidatorInfo): Promise<void> {
```

Comment:
Is upsert a common term? Does it convey something different than "update"?

Reply:
See my other comment

Reply:
(And yeah, it's pretty common at least in the SQL DB world.)